DTS-STN/canadian-dental-care-plan
# Removing non-injectable service and mapper in favour of injectable ones

### Description
- Changes as a result of:
  - #2507
  - #2531
  - #2545
- Moved and renamed `benefit-application-service-schemas` into the `mocks` directory since it is only used there

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/9e6bab13-8d64-49ec-9ebd-9a5931e63bc3)

### Checklist

### Test Instructions

- Run the application and complete an application with any or all of the `adult`, `adult-child`, `child` flows. You should be able to submit an application without issues. 
- Feel free to also perform a payload comparison on the "Review" page with the CDCP performance environment.
